src/web/mjs/engine/Storage.mjs

```diff
@@ -911,4 +912,20 @@ export default class Storage {
             } );
         } );
     }
+
+    get downloadedConnectors() {
+        //since this is a async function, isn't hakuneko startup waiting for this to be ran first?
+        //seems like performance drain
+        let rootDirectoryEntries = this._readDirectoryEntries(this.root);
+        let arrayFiltered = [];
+        rootDirectoryEntries.then(function(arrayRaw){
+            arrayRaw = arrayRaw.filter(plugin => plugin.startsWith('hakuneko.mangas.'));
+            arrayRaw.forEach(file => {
+                arrayFiltered.push(file.substr(file.indexOf('.', 15)+1));
+            });
+            return arrayFiltered;
+        })
```

Comment:
`electron.remote.app` was not working so decided on the new variable `this.root` here.
retrieved the directory with `_readDirectoryEntries` as Ronny surgested